game/game_logic.rb

```diff
@@ -0,0 +1,46 @@
+module GameLogic
+  DIFFICULTY_HASH = {
+    I18n.t(:easy) => [15, 3],
+    I18n.t(:medium) => [10, 2],
+    I18n.t(:hard) => [5, 1],
+    I18n.t(:hell) => [3, 0]
+  }.freeze
+
+  def make_number(chars = 4, numbers = 6)
+    (1..chars).map { rand(1..numbers) }.join
+  end
+
+  def check_numbers(secret, numbers)
```

Comment:
Could you please find a better solution for checking number? Passing an array twice is not an effective solution.